cli/cmd/policy_update.go

```diff
@@ -150,3 +181,41 @@ func updatePolicy(cmd *cobra.Command, args []string) error {
 	cli.OutputHuman("The policy %s was updated.\n", updateResponse.Data.PolicyID)
 	return nil
 }
+
+func policyBulkUpdate(args []string) error {
+	var (
+		policyIds []string
+		err       error
+	)
+	// if no policy ids are provided; prompt a list of policy ids
+	if len(args) == 0 {
+		policyIds, err = promptSetPolicyIDs()
+		if err != nil {
+			return err
+		}
+	} else {
+		policyIds = args
+	}
+
+	if !lwseverity.IsValid(policyCmdState.Severity) {
```

Comment:
Would this make better sense at line 125?

Reading through the code, I didn't expect to see validation at this point along the code path.

Reply:
👍